deeppavlov/dataset_readers/snips_reader.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2019 Neural Networks and Deep Learning lab, MIPT
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from typing import List, Dict, Any, Optional
+from pathlib import Path
+
+from overrides import overrides
+
+from deeppavlov.core.common.registry import register
+from deeppavlov.core.data.dataset_reader import DatasetReader
+
+
+@register('snips_reader')
```

Comment:
Registered classes should be added to the [registry.json](https://github.com/Huawei-MRC-OSI/DeepPavlov/blob/snips_dataset/deeppavlov/core/common/registry.json) file by hand or using the [utils.prepare.registry](https://github.com/Huawei-MRC-OSI/DeepPavlov/blob/snips_dataset/utils/prepare/registry.py) script that will, however, require all the local configs requirements

Reply:
Fixed.